tests/FINDOLOGIC/Export/Tests/DataElementsTest.php

```diff
@@ -198,4 +198,16 @@ public function testUsergroupStringRepresentationIsTheUsergroupValue()
 
         $this->assertEquals($usergroup->getValue(), (string) $usergroup);
     }
+
+    /**
+     * @expectedException \FINDOLOGIC\Export\Helpers\AttributeValueLengthException
+     */
+    public function testVeryLongAttributeValueCausesException()
+    {
+        $attribute = new Attribute('attribute_with_very_long_value');
+
+        $value = implode('', array_fill(0, 16384, 'a'));
```

Comment:
This test should break if `checkAttributeValueNotExceedingCharacterLimit` uses `strlen()` instead of `mb_strlen()`, but it won't, because `a` only takes one byte. Use a multibyte character in this test instead.

Reply:
Done :heavy_check_mark: